libs/wasm-transform/examples/usage.rs

```diff
@@ -26,8 +29,13 @@ fn overwrite() {
 
     let req = transform::Request {
```

Comment:
Let's stick a quick comment explaining what this does: "Transform rule to transform `{"hello": "baz"}` to `{"hello": ""}`.